docs/2.Basics/Suppressing and Skipping Policies.md

```diff
@@ -94,3 +94,59 @@ Resources:
       MasterUserPassword: 'password' # checkov:skip=CKV_SECRET_6 or next to it
       # checkov:skip=CKV_SECRET_6 or after it
 ```
+
+## CloudFormation Metadata
+Additionally, it is possible to suppress CloudFormation checks via the `Metadata` section inside a resource.
+```yaml
+Resources:
+  MyDB:
+    Metadata:
+      checkov:
+        skip:
+          - id: "CKV_AWS_157"
+            comment: "Ensure that RDS instances have Multi-AZ enabled"
+    Type: "AWS::RDS::DBInstance"
+    Properties:
+      DBName: "mydb"
+      DBInstanceClass: "db.t3.micro"
+      Engine: "mysql"
+      MasterUsername: "master"
+      MasterUserPassword: "password"
+```
+
+### CDK Example
+The `Metadata` section of a CDK construct can only be adjusted via the L1 resource.
```

Comment:
what is L1?

Reply:
this a term used in CDK docs, it stands for Layer 1 or also called Level 1. These are the low level CloudFormation resources. There also L2 and even L3 constructs, which are abstractions of the low level CloudFormation resources.